src/sass/app/websocket_pages.scss

```diff
@@ -283,3 +283,9 @@
         justify-content: center;
     }
 }
+
+.realws-content {
+    div.select-dropdown {
+        width: 100%;
```

Comment:
we have some very long state names, or country names, I can't remember which. sometimes 100% causes it to become too long. please check this,

Reply:
Yeah, countries, this doesn't affect that one. Thanks for pointing that out. I'll double check the mobile ones.